components/host-dashboard/reports-section/TransactionsOverviewSection.js

```diff
@@ -18,6 +19,11 @@ import { formatAmountForLegend } from './helpers';
 // Dynamic imports
 const Chart = dynamic(() => import('react-apexcharts'), { ssr: false });
 
+const StyledBox = styled(Box)`
+  .apexcharts-toolbar {
+    z-index: 0 !important;
```

Comment:
Seems the `!important` isn't needed? 

```suggestion
    z-index: 0;
```

Reply:
Thanks for the feedback. Let me make changes